index.html

```diff
@@ -15,20 +15,25 @@
         <div class="container">
           <nav class="navbar flex">
             <div class="logo">
-              <a href="index.html"><h2>My Logo</h2></a>
+              <h2>My Logo</h2>
             </div>
             
-            <div class='menu-list'>
-              <ul>
-                <li><a href="#work-station" aria-label ="portfolio area"></a>Portfolio</li>
-                <li><a href="#about-station" aria-label ="about me area"></a>About</li>
-                <li><a href="#contact-station" aria-label ="contact me area"></a>Contact</li>
-              </ul>
-            </div>
-
+            <ul class="navnav-menu">
+              <li>
+                <a href="#work-section" class="navnav-items" style="text-decoration: none;">Portfolio</a>
+              </li>
+              <li>
+                <a href="#about-section" class="navnav-item" style="text-decoration: none;">About</a>
+              </li>
+              <li>
+                <a href="#contact-section" class="navnav-item" style="text-decoration: none;">Contact</a>
+              </li>
+            </ul>
```

Comment:
Currently, when I run your project, the hamburger menu is showing which is not the expectation according to the Figma template. Kindly make it disappear and let the links be visible on the desktop and the hamburger on the mobile in order to be consistent with Figma.

Your Header

![Badheading](https://user-images.githubusercontent.com/84930355/223435491-69314b4e-8f97-4091-bea8-19f53b04a37a.PNG)

FIGMA

![Goodheading](https://user-images.githubusercontent.com/84930355/223436160-3e4538cb-f5c2-4a3c-950d-5f4c0fc08996.PNG)